specification/BUILDING.md

```diff
@@ -0,0 +1,110 @@
+<!-- omit in toc -->
```

Comment:
I noticed that the 3D Tiles logo was small in `Specification-1.1.0-compressed-2022-06-26.pdf`. I didn't see this problem when I generated the PDF myself.

![small-logo](https://user-images.githubusercontent.com/915398/170699809-f5c05770-abd9-49b8-b5ef-610a8cb0d82d.png)

Reply:
Strange, I have some PDF where this was still the right size, but at some point, that chnaged - will take a note